### PR TITLE
Division added 

### PR DESCRIPTION
**Pull request by TheXorcist**
doya kore check korun

### CODE REVIEW

**Review by CaptainRohan — APPROVED** [review]
hoyeche onek korecho

**Review by J-Caesar2000 — APPROVED** [review]
Don't divide, just unite!

File: Components/Division.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+namespace Components.Division
+{
+    
+    class Division
+    {
+        public static float Devide(int a, int b)
```
Comment (SubhamK108) [inline Components/Division.cs]: Make it Int

**Review by SubhamK108 — CHANGES_REQUESTED** [review]

**Review by SubhamK108 — APPROVED** [review]

File: Components/Division.cs (hunk 1 above)
Comment (CaptainRohan) [reply Components/Division.cs]: fuck you

**Pull request merged**